app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -289,14 +294,30 @@ class HomeMenu(
             onItemTapped.invoke(Item.Extensions)
         }
 
-        val syncSignInItem = BrowserMenuImageText(
-            context.getString(R.string.library_synced_tabs),
+        val syncedTabsItem = BrowserMenuImageText(
+            context.getString(R.string.synced_tabs),
             R.drawable.ic_synced_tabs,
             primaryTextColor
         ) {
             onItemTapped.invoke(Item.SyncTabs)
         }
 
+        val syncItemTitle =
+            if (accountManager.accountProfile()?.email != null) {
+                signedInToFxA = true
+                accountManager.accountProfile()?.email!!
```

Comment:
You can avoid the !! by setting `accountManager.accountProfile()?.email` to a variable
```suggestion
        val email = accountManager.accountProfile()?.email
        val syncItemTitle = email ?: context.getString(R.string.sync_menu_sign_in)
```